tilingManager/tilingLayouts/baseTiling.js

```diff
@@ -240,4 +241,8 @@ var BaseTilingLayout = class BaseTilingLayout {
             !metaWindow.resizeable
         );
     }
+
+    setTitleBarVis(window, vis) {
+        new NoTitleBarModule().setWindowTitleBarVis(window, vis);
```

Comment:
A module should only be initiated once since it's need to be disabled when the extension is disabled. If you want to exernalise a function please create an utils file and use it in the module and in the files you need to call it.